build/dependency

```diff
@@ -25,23 +25,54 @@ Examples:
 EOF
 }
 
+pkg_chdir() {
+    #
+    # Note: we must have three layers of directory because old autotools
+    # misbehave when we are using only two layers and specifically they try
+    # to use measurement-kit's autotools rather than their own.
+    #
+    # See measurement-kit/measurement-kit#1017 build history for more.
+    #
+    cd ./build/third_party/$pkg_name
+}
+
 pkg_autogen() {
     if [ -x ./autogen.sh ]; then
-        echo "- ./autogen.sh"
-        ./autogen.sh
+        echo "- ./autogen.sh $pkg_autogen_sh_flags"
+        ./autogen.sh $pkg_autogen_sh_flags
     elif [ -f ./configure.ac ]; then
         echo "- autoreconf -i"
         autoreconf -i
     fi
 }
 
+pkg_depend() {
+    for dep in `echo $pkg_deps`; do
+        if [ "$pkg_cross" != "" -a "$pkg_arch" != "" ]; then
+            dep="$pkg_cross-$pkg_arch-$dep"
+        fi
+        if [ ! -f $pkg_rootdir/$dep.tgz ]; then
+            baseurl=https://github.com/measurement-kit/dependencies
+            wget -O $pkg_rootdir/$dep.tgz                                      \
+                $baseurl/releases/download/2016-12-19/$dep.tgz
```

Comment:
Possibly. However we may want to have different dependencies in different branches.